docs/GettingStartedOnLinux.md

```diff
@@ -0,0 +1,171 @@
+---
+id: getting-started
```

Comment:
You'll need to change the id to `getting-started-linux`. Also need to edit the `getting-started` page so it includes link to this page.